python/dbsp/config.py

```diff
@@ -70,6 +70,21 @@ def add_file_input(self, stream: str, filepath: str, format_: FormatConfig, **fi
                         name = "file",
                         config = FileInputConfig.from_dict(dict({ 'path': filepath }, **file_config_options))),
                     format_ = format_))
+    
```

Comment:
These Python API changes will need to be aligned with @gz's PR.

Reply:
I'll wait on him to merge his PR and then rebase.